src/Pages/ProductLists/Components/CategoryItems.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component, Fragment } from "react";
+import { APIYJ } from "../../../config";
+
+class CategoryItems extends Component {
+  constructor() {
+    super();
+    this.state = {
+      categoryItems: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch(`${APIYJ}`)
+      .then((res) => res.json())
+      .then((res) => this.setState({ categoryItems: res.CategoryItems }));
+  }
+
+  render() {
+    const { categoryItems } = this.state;
+    return (
+      categoryItems.length > 0 && (
+        <Fragment>
+          {categoryItems.map((item) => {
+            return <div className="categoryItems">{item.CategoryItem}</div>;
+          })}
+        </Fragment>
+      )
+    );
```

Comment:
이렇게 줄일 수 있겠네요!
Fragment 활용 굿!
```suggestion
    return (
        <Fragment>
          {categoryItems.length > 0 && categoryItems.map((item) => {
            return <div className="categoryItems">{item.CategoryItem}</div>;
          })}
        </Fragment>
    );
```

Reply:
넵 수정했습니다~